conda/environments/cudf_dev_cuda10.0.yml

```diff
@@ -46,6 +46,7 @@ dependencies:
   - rapidjson
   - flatbuffers
   - hypothesis
+  - spdlog
```

Comment:
Need to add this to the conda recipe. Or preferably, just remove it altogether from the env files because you added the dependency to RMM here: https://github.com/rapidsai/rmm/pull/362

Reply:
Done.